use-shopping-cart/src/@types/use-shopping-cart/index.d.ts

```diff
@@ -0,0 +1,105 @@
+declare module 'use-shopping-cart' {
+
+    export interface ProviderProps {
+        children: JSX.Element,
+        stripe: any,
```

Comment:
Stripe should not be of type any, it should be of type `stripe.Stripe` which is coming from `@types/stripe-v3`

Reply:
Yeah i saw that but wasn't sure if i could just use it as i cant import it. I think it works just to state it though as those types are installed in the package.